README.md

```diff
@@ -55,8 +55,8 @@ and other RAPIDS packages.
 
 ### CUDA/GPU requirements
 
-* CUDA 11.2+
-* NVIDIA driver 450.80.02+
+* CUDA 12.0+
+* NVIDIA driver 525.60.13+
```

Comment:
The build team has said for a long time that we want to stop listing driver requirements and instead do this:
```suggestion
* CUDA 12.0+ with a compatible NVIDIA driver
```